src/ToDo.js

```diff
@@ -11,23 +11,25 @@ class ToDo extends React.Component {
     this.state = {
       tasks: {},
       countItems: 0,
-      completed: {}
+      completed: {},
     };
   };
 
-  showValue = (event) => {
-    if (event.key === 'Enter') {
-      if (event.target.value && this.state.countItems !== 6) {
-        let eventValue = event.target.value;
-        event.target.value = '';
-        this.setState(prevState => {
-          let copied = { ...prevState.tasks };
-          let countTaks = this.countCompletedTasks(this.state.tasks)
-          copied[eventValue] = false;
-          prevState.countItems = countTaks + 1;
-          return { tasks: copied };
+  addTask = (event) => {
+    if (event.key !== 'Enter') {
```

Comment:
Wrap your text input with `<form onSubmit={addTask} ... >` and you won't need to handle this `keyPress` event